src/ConnectionTest.php

```diff
@@ -201,9 +201,14 @@ protected function render_admin_page() {
 							<!--<pre><?php var_dump( $user_token ); ?></pre>-->
```

Comment:
Unrelated to this PR but do we ever need this commented `var_dump`? Seems to be added for debugging purposes and forgotten.

Reply:
I was using it in the initial stages, to test authenticated API requests, and purposely left it there in case I needed to do so later down the line. But looking at it again, we can remove it, as there are other ways to build an authenticated request.